backend/Piece.py

```diff
@@ -0,0 +1,87 @@
+from enum import Enum
+
+
+class Turn(Enum):
```

Comment:
I feel like this could be passed into the Piece class from a different package but Im not sure about if that architecture makes more sense.

Reply:
To me Turn is more cohesive in the GameManager package.

Reply:
I don't think Turn should be in the GameManager, even if it's more cohesive, because I'm calling it in the Pieces class. It can be in its own separate class, but we should not import anything from GameManager into the Piece.py file.

I guess I can redo how Piece colors are stored, if we'd prefer that -- Turn would be fine to move to GameManager then.